apps/web/src/api/FaucetApi.ts

```diff
@@ -0,0 +1,49 @@
+import { NetworkConnection } from "@contexts/Environment";
+import { getBaseUrl } from "@api/index";
+
+export default {
+  sendFundsToUser: async (
+    network: NetworkConnection,
+    recaptchaValue: string,
+    contractAddressHash: string,
+  ): Promise<FaucetTransactionResponse> => {
+    const baseUrl = getBaseUrl();
+    const res = await fetch(
+      `${baseUrl}faucet/${contractAddressHash}?network=${network}`,
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          recaptchaValue,
+        }),
+      },
+    );
+    return res.json();
+  },
+};
+
+export interface FaucetTransactionResponse {
```

Comment:
this is TransactionResponse you can get it from ethers lib

Reply:
import {  TransactionReceipt } from 'ethers';

Reply:
Are you referring to this? `import { TransactionRequest } from "@ethersproject/providers";`?

Reply:
TS2305: Module  "ethers"  has no exported member  TransactionReceipt 

I dont think it will fulfil the props that are needed though